tools/blast_parser/conf/blast_parser_wrapper.xml

```diff
@@ -0,0 +1,36 @@
+<tool id="blast_parser_wrapper" name="blast_parser" version="0.1.1">
+	<description>
+		Parse 12 column BLAST result into 3 column hcluster_sg input
+	</description>
+	 <requirements>
+        <requirement type="package" version="0.1.1">blast_parser</requirement>
```

Comment:
You don't need this requirement, since the script will be provided together with the XML wrapper.